src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -518,7 +518,13 @@ private function createEncoder($config, ContainerBuilder $container)
         // Argon2i encoder
         if ('argon2i' === $config['algorithm']) {
             if (!Argon2iPasswordEncoder::isSupported()) {
-                throw new InvalidConfigurationException('Argon2i algorithm is not supported. Please install the libsodium extension or upgrade to PHP 7.2+.');
+                $version = explode(".", phpversion());
+                if(7 === $version[0] && $version[1] >= 2)
```

Comment:
`if (\PHP_VERSION_ID >= 70200) {` (we don't add line breaks before opening brackets)